recipe/meta.yaml

```
@@ -20,31 +22,35 @@ build:
     - sphinx-autogen = sphinx.ext.autosummary.generate:main
 
 requirements:
+  build:
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
     - python
     - pip
     - flit-core >=3.7
```

```suggestion
    - flit-core >=3.11
```
https://github.com/sphinx-doc/sphinx/blob/v8.2.3/pyproject.toml#L2